app/merginapi.h

```diff
@@ -5,37 +5,109 @@
 #include <QNetworkAccessManager>
 #include <QEventLoop>
 #include <memory>
+#include <QFile>
+
+enum ProjectStatus
+{
+  NoVersion,
+  UpToDate,
+  OutOfDate
+};
+Q_ENUMS( ProjectStatus )
 
 struct MerginProject {
     QString name;
     QStringList tags;
-    QString info;
+    QString created;
+    QString updated; // local updated
+    QString serverUpdated;
+    bool pending = false;
```

Comment:
pending means that we have a pending download/update request in MerginApi for this project right? Can we add a short comment about what does it mean?